Dockerfile

```diff
@@ -1 +1,47 @@
 FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_arkcase_core \
+      MAINTAINER="Armedia LLC"
+
+ARG ARKCASE_VERSION=2021.02
+ARG TOMCAT_VERSION=9.0.50 
+ARG TOMCAT_MAJOR_VERSION=9
+
+ENV NODE_ENV="production" \
+    ARKCAE_APP="/app/arkcase" \
+    JAVA_OPTS="$JAVA_OPTS -Djava.net.preferIPv4Stack=true -Duser.home=${ARKCAE_APP}/data/arkcase-home" \
+    TMP=/app/arkase/tmp \
+    TEMP=/app/arkase/tmp \
+    PATH=$PATH:/app/tomcat/bin
+WORKDIR /app
+COPY server.xml ./
+# ADD yarn repo and nodejs package
+ADD https://dl.yarnpkg.com/rpm/yarn.repo /etc/yum.repos.d/yarn.repo
+ADD https://archive.apache.org/dist/tomcat/tomcat-${TOMCAT_MAJOR_VERSION}/v${TOMCAT_VERSION}/bin/apache-tomcat-${TOMCAT_VERSION}.tar.gz /app
+RUN useradd --system --user-group --no-create-home tomcat && \
+    mkdir -p ${ARKCAE_APP}/data/arkcase-home && \
+    mkdir -p ${ARKCAE_APP}/common &&\
+    yum --assumeyes update; \
+    # Nodejs prerequisites to install native-addons from npm
+    yum install --assumeyes gcc g++ make openssl wget zip unzip && \
+    curl –sL https://rpm.nodesource.com/setup_6.x | bash - && \
+    yum install --assumeyes nodejs && \
+    #unpack tomcat tar to tomcat directory
+    tar -xf apache-tomcat-${TOMCAT_VERSION}.tar.gz && \
+    mv apache-tomcat-${TOMCAT_VERSION} tomcat && \
+    rm apache-tomcat-${TOMCAT_VERSION}.tar.gz &&\
+    # Removal of default/unwanted Applications
+    rm -rf tomcat/webapps/* tomcat/temp/* tomcat/logs tomcat/bin/*.bat && \
```

Comment:
If you are removing the logs directory then you have to add the logging.properties file (see https://github.com/ArkCase/ark_alfresco/blob/develop/artifacts/logging.properties) that we are using to stop logging to logs and you need to prevent the access logs from going to the logs directory as well.

Reply:
Thanks @ymuwakki . I have followed same you have did for alfresco. I hope this will redirect all the access logs and application logs to console directly.